packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -13,12 +13,26 @@ void main() {
 
   const String longText = 'one two three four five six seven eight nine ten eleven twelve';
   final List<DropdownMenuEntry<TestMenu>> menuChildren = <DropdownMenuEntry<TestMenu>>[];
+  final List<DropdownMenuEntry<TestMenu>> menuChildrenWithIcons = <DropdownMenuEntry<TestMenu>>[];
 
   for (final TestMenu value in TestMenu.values) {
     final DropdownMenuEntry<TestMenu> entry = DropdownMenuEntry<TestMenu>(value: value, label: value.label);
     menuChildren.add(entry);
   }
 
+  ValueKey<String> leadingIconKey(TestMenu menuEntry) => ValueKey<String>(' leading-${menuEntry.label}');
+  ValueKey<String> trailingIconKey(TestMenu menuEntry) => ValueKey<String>(' trailing-${menuEntry.label}');
```

Comment:
Just out of curiosity, why the leading spaces in the ValueKey strings here?

Reply:
Good catch @justinmc! No reason for these spaces, just a typing error that did not get noticed.